src/test/java/seedu/address/testutil/TypicalPersons.java

```diff
@@ -27,14 +27,19 @@ public class TypicalPersons {
             .withPhone("94351253").withLessonPlan("Algorithms")
             .withTags("friends").build();
```

Comment:
hello sir u need to add ur attendance here also cos u added in the typicalPersonsAddressBook.json and they have to match

Reply:
hehe yes tried that first *cries and smiles*